build.gradle.kts

```diff
@@ -39,7 +39,7 @@ apply {
     from(file("gradle/compile.gradle.kts"))
 }
 
-android {
+configure<com.android.build.gradle.AppExtension> {
```

Comment:
@eskatos We can no longer use `android`?

Reply:
Not if you are not using the `plugins {}` block.
See https://docs.gradle.org/current/userguide/kotlin_dsl.html#type-safe-accessors